Gems/Atom/RPI/Code/Include/Atom/RPI.Public/AssetQuality.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/base.h>
+
+namespace AZ
+{
+    namespace RPI
+    {
+        using AssetQuality = u32;
+
+        constexpr AssetQuality AssetQualityHighest = 0;
+        constexpr AssetQuality AssetQualityLowest = 0xFFFFFFFF;
+    }
+}
```

Comment:
consider consolidating the namespace lines into one for brevity

```suggestion
namespace AZ::RPI
{
    using AssetQuality = u32;

    constexpr AssetQuality AssetQualityHighest = 0;
    constexpr AssetQuality AssetQualityLowest = 0xFFFFFFFF;
}
```

Reply:
We didn't use this kind of namespace for other files in RPI library. I think it's okay to keep it this way for consistency.